openbb_terminal/common/technical_analysis/ta_helpers.py

```diff
@@ -0,0 +1,42 @@
+from typing import Optional
+
+import pandas as pd
+
+from openbb_terminal.rich_config import console
+
+
+def check_columns(
+    data: pd.DataFrame, high: bool = True, low: bool = True, close: bool = True
+) -> Optional[str]:
+    """Returns the close columns, or None if the dataframe does not have required columns
+
+    Parameters
+    ----------
+    data: pd.DataFrame
+        The dataframe to check
+    high: bool
+        Whether to check for high column
+    low: bool
+        Whether to check for low column
+    close: bool
+        Whether to check for close column
+
+    Returns
+    ----------
+    Optional[str]
+        The name of the close column, none if df is invalid
+
+    """
+    close_col = "Adj Close" if "Adj Close" in data.columns else "Close"
```

Comment:
Another potential edge case: adj_close/Adj_Close

Reply:
We arent doing any extra handling for other columns. I say we just continue printing it as it is and make them change column names, but if you disagree I can change.

Reply:
I think we can just throw in those (mostly because qa menu maayyyyy mess with the loaded df).

Reply:
Cool. Done.